### PR TITLE
Decrease frequency of dependabot alerts to monthly

### PR DESCRIPTION
**Pull request by zricethezav**
Title says it all. Automatically updating all go mod dependencies every week is too much

### CODE REVIEW

File: .github/dependabot.yml
Comment (rosecodym) [inline .github/dependabot.yml:15]: Should we update this comment too? Or better yet, delete it?

**Review by rosecodym — APPROVED** [review]
While I approve of this in spirit I don't know why the original frequency was picked so you should probably wait for the approval of someone who does :v

**Review by mcastorina — APPROVED** [review]
I can't say for certain why a weekly cadence was chosen - I think it was one of those quality of life automations that Dustin setup to make sure our dependencies are up to date.

Monthly cadence is good for the amount of PRs it's generating now imo.

**Pull request merged**
